libs/core/src/components/pds-accordion/test/pds-accordion.spec.tsx

```diff
@@ -35,7 +35,7 @@ describe('pds-accordion', () => {
             <details open>
               <summary>
                 <slot name="label">Details</slot>
-                <pds-icon name="up-small"></pds-icon>
+                <pds-icon icon="data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' class='pdsicon'><path fill-rule='evenodd' d='M7.15 5.594a.889.889 0 0 1 1.256 0l3.556 3.555a.889.889 0 0 1-1.257 1.257L7.778 7.48 4.85 10.406A.889.889 0 1 1 3.594 9.15l3.555-3.555Z'/></svg>"></pds-icon>
```

Comment:
in the event that the svg changes ever so slightly, I would look to change this to read from the actual imported variable. see snippet below


```
import { downSmall, upSmall } from '@pine-ds/icons/icons';
...
<pds-icon icon={downSmall} ....
```